mass_mailing_editor_af/views/snippets_themes.xml

```diff
@@ -1186,6 +1185,22 @@
         </div>
     </template>
 
+    <!--textmodul med H3 och brödtext med check-ikon-->
+    <template id="s_mail_block_toppyta_h3_checkikon" name="H3 och brödtext med check-ikon">
+        <div class="snippets_div" style="max-width: 600px;background-color: white; padding: 8px 24px 0px 34px;">
+            <img src="/mass_mailing_editor_af/static/src/img/blocks/ikoncheck.svg" style="width: 26px;height: 20px; margin-top:10px"/>
+            <div style="display:inline-block ; width:260px; background-color: white; vertical-align: top; margin-left:34px">
+                <h3 class="snippets_h1_dark" style="width: 482px;color: rgb(51, 51, 51);
```

Comment:
remove width: 260px to be able to get responsive design

Reply:
Why is this class used? (snippets_h1_dark) Should the font-size increase when the screen gets smaller? If not, remove this class